aptos-move/framework/src/natives/aggregator_natives/context.rs

```diff
@@ -164,31 +174,73 @@ mod test {
     // All aggregators are initialized deterministically based on their ID,
     // with the following spec.
     //
-    //     +-------+---------------+----------+-----+---------+
-    //     |  key  | storage value |  create  | get | remove  |
-    //     +-------+---------------+----------+-----+---------+
-    //     |  100  |               |   yes    | yes |   yes   |
-    //     |  200  |               |   yes    | yes |         |
-    //     |  300  |               |   yes    |     |   yes   |
-    //     |  400  |               |   yes    |     |         |
-    //     |  500  |               |          | yes |   yes   |
-    //     |  600  |               |          | yes |         |
-    //     |  700  |               |          | yes |         |
-    //     |  800  |               |          |     |   yes   |
-    //     +-------+---------------+----------+-----+---------+
-    fn test_set_up(context: &NativeAggregatorContext) {
+    //     +-------+---------------+-----------------------------------+-----+---------+
+    //     |  key  | storage value |  create                           | get | remove  |
+    //     +-------+---------------+-----------------------------------+-----+---------+
+    //     |  100  |               |   yes                              | yes |   yes   |
+    //     |  200  |               |   yes                              | yes |         |
+    //     |  300  |               |   yes                              |     |   yes   |
+    //     |  400  |               |   yes                              |     |         |
+    //     |  500  |               |   Yes if !aggregagtor_enabled      | yes |   yes   |
+    //     |  600  |               |   Yes if !aggregagtor_enabled      | yes |         |
+    //     |  700  |               |   Yes if !aggregagtor_enabled      | yes |         |
+    //     |  800  |               |                                    |     |   yes   |
+    //     +-------+---------------+------------------------------------+-----+---------+
```

Comment:
You don't need to do `Yes if !aggregagtor_enabled `: all you need it to add some data to `EmptyStorage` you can see for example how this can be done here:

https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/aptos-aggregator/src/aggregator_extension.rs#L381

and you can add data to `FakeDataStore`. Like here:

https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/aptos-aggregator/src/delta_change_set.rs#L581

Reply:
Done.